models/Group.py

```diff
@@ -0,0 +1,21 @@
+"""User Model"""
+from bson.objectid import ObjectId
+from mongoengine.base.fields import ObjectIdField
+from mongoengine.document import Document
+from mongoengine.fields import ListField, StringField
+
+# pylint: disable=no-member
+class Group(Document):
+    """Group Collection"""
+
+    _id = ObjectIdField(default=ObjectId(), primary_key=True)
```

Comment:
Maybe try changing default=ObjectId() to default=ObjectId, else we'll get problems when updating/posting new documents. Since if ObjectId() doesn't change due to fast or consecutive document upsert/insert we'll be replacing the last document in the db.

```suggestion
    _id = ObjectIdField(default=ObjectId, primary_key=True)
```